app/layout.tsx

```diff
@@ -1,26 +1,38 @@
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import type {Metadata} from "next";
+import {Inter} from "next/font/google";
 import React from "react";
 import showdown from "showdown";
-import { getInstaller } from "@/common";
-import { Link, PoweredByNuon } from "@/components";
+import {getInstaller} from "@/common";
+import {Link, PoweredByNuon} from "@/components";
 import "./globals.css";
 
 const markdown = new showdown.Converter();
-const inter = Inter({ subsets: ["latin"] });
+const inter = Inter({subsets: ["latin"]});
 
-export const metadata: Metadata = {
-  title: "Nuon installers",
-  description:
-    "Let your customers install your app in their cloud account with only a few clicks",
+export async function generateMetadata(
+): Promise<Metadata> {
+  const {metadata} = await getInstaller();
```

Comment:
I think Next will cache the result, so it doesn't get fetched multiple times if we call this from multiple templates.